### ISSUE TITLE
Developer API like chunky?

### ISSUE THREAD

**Issue by kofeychi** [pop4959/ChunkyBorder#84]
gradle does not want me to use the jar of chunky border

**Comment by pop4959** [comment]
Yes a simple API is something that I have planned to add. However you should still be able to use the implementation dependency, in exactly the same way as documented for the chunky API on the wiki using maven (just changing out chunky for chunky border). You should not use the jar file directly.
